andela/AndelaEatsUI
# #161148866 Add a nice no-page-content UI

#### What does this PR do?
- Adds user friendly and presentable UI for no page content displays

#### Description of Task to be completed?
- Add global `empty-content` class
- Update no-content section on menus page

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ch-update-no-content-ui-161148866`
- Run these commands `npm run bundle` and `npm run dev` to setup app
- Log in as an `Admin`, navigate to different page to find that with no content
- Ensure the a nice looking UI is seen.

#### What are the relevant pivotal tracker stories?
- [#161148866](https://www.pivotaltracker.com/story/show/161148866)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-11 at 3 07 03 pm" src="https://user-images.githubusercontent.com/32123313/46809864-5703ea00-cd67-11e8-92e2-1cb6f067867b.png">
<img width="1440" alt="screen shot 2018-10-12 at 4 17 09 pm" src="https://user-images.githubusercontent.com/32123313/46878410-268d8000-ce3b-11e8-925f-20357e1196f9.png">
<img width="1440" alt="screen shot 2018-10-12 at 4 17 58 pm" src="https://user-images.githubusercontent.com/32123313/46878411-268d8000-ce3b-11e8-9b46-1f6044493b35.png">
<img width="1440" alt="screen shot 2018-10-12 at 4 19 37 pm" src="https://user-images.githubusercontent.com/32123313/46878412-27261680-ce3b-11e8-8602-567cdce9adb1.png">
<img width="1440" alt="screen shot 2018-10-12 at 4 22 29 pm" src="https://user-images.githubusercontent.com/32123313/46878413-27261680-ce3b-11e8-925d-f3407530246c.png">
